Website/DesktopModules/Admin/Security/EditUser.ascx

```diff
@@ -10,14 +10,33 @@
 
 <%@ Import Namespace="DotNetNuke.UI.Utilities" %>
 
+<%
+  var confirmText = ClientAPI.GetSafeJSString(LocalizeString("UnregisterUser"));
+  if (PortalSettings.DataConsentActive && User.UserID == UserInfo.UserID)
```

Comment:
Hi @donker, can you please explain what the `User.UserID == UserInfo.UserID` condition is for? I see it down below as well and it seems that `PortalSettings.DataConsentActive` can be enabled but this other condition would defeat that setting and revert back to previous behavior

Reply:
I'm guessing it's to only allow the User to request their account is removed. However, I'll wait to merge till @donker can give some more details.

Reply:
I wanted to be absolutely sure you cannot get here while editing someone else's profile. There is a lot of reuse of the profile editing controls and I was afraid it might not be the editing user who is deleting. In that case it shouldn't be marked as a request so the old logic is used.

Reply:
Cool, so using old logic in such case was the intended purpose.